usmqe/api/tendrlapi/cephapi.py

```diff
@@ -0,0 +1,14 @@
+"""
+Tendrl REST API for ceph.
+"""
+
+import pytest
+from usmqe.api.tendrlapi.common import TendrlApi
+
+LOGGER = pytest.get_logger("tendrlapi_ceph", module=True)
+
+
+class TendrlApiCeph(TendrlApi):
+    """ Ceph methods for Tendrl REST API.
+    """
+    pass
```

Comment:
I suggest to change the `import_cluster` method here and in the gluster api too. It will automatically add `sds_pkg_name` to the method parameters.